tools/proteomiqon_psmbasedquantification/proteomiqon_psmbasedquantification.xml

```diff
@@ -0,0 +1,449 @@
+<tool id="proteomiqon_psmbasedquantification" name="Proteomiqon PSMBasedQuantification" version="@VERSION@" profile="20.05">
+    <description>
+        allows label-free quantification as well as quantification of full metabolic labeled samples.
+    </description>
+    <macros>
+        <token name="@VERSION@">0.0.7</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">proteomiqon-psmbasedquantification</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #set instrumentOutput_basename = $re.sub(r'[^\w ,.\-+]','_',$instrumentOutput.element_identifier)
+        #set scoredPSMs_basename = $re.sub(r'[^\w ,.\-+]','_',$scoredPSMs.element_identifier)
```

Comment:
Since you are in control of the program, do you have any smart idea how we can avoid encoding metadata in file names that we need to preserve?

Reply:
The tool doesn't actually need this. It is just a convenience for the user to see the original file names and improve readability in the output